daemon/algod/api/algod.oas2.json

```diff
@@ -1365,6 +1366,10 @@
           "description": "\\[algo\\] total number of MicroAlgos in the account",
           "type": "integer"
         },
+        "min-balance": {
+          "description": "MicroAlgo balance required by the account.\n\nThe requirement starts at 100,000 and grows based on asset and application usage.",
```

Comment:
I think it would be better to drop off the numeric constants from the description. These values are consensus protocol dependent and can be modified in the future.

Reply:
👍